man/en/sos-collect.1

```diff
@@ -258,8 +260,17 @@ are not used.
 If \fB\-\-preset\fR is specified and a given node either does not have that preset
 defined, or has a version of sos prior to 3.6, this option is ignored for that node.
 .TP
-\fB\-p\fR SSH_PORT, \fB\-\-ssh\-port\fR SSH_PORT
-Specify SSH port for all nodes. Use this if SSH runs on any port other than 22.
+\fB\-\-skip-commands\fR COMMANDS
+A comma delimited list of commands to skip execution of, but still allowing the
+rest of the plugin that calls the command to run. This will generally need to
+be some form of UNIX shell-style wildcard matching. For example, using a value
+of \fBhostname\fR will skip only that single command, while using \fBhostname*\fR
+will skip any any all hostname commands.
```

Comment:
"any any" sounds a typo for me, non-native English speaker..?

Reply:
Yup, that should be "any and".

Reply:
How about something like:

```
while using \fBhostname*\fR will skip all commands with names that begin with the string "hostname".
```

To more accurately describe the pattern match for that string? (I know we all get it, but to a user who's less familiar with typical UNIX/Linux conventions the phrase "any and all hostname commands" might imply that it would exclude `oshostname` or `whatsthehostname` when the example pattern will not).

Reply:
+1 for the "names that begin".

Reply:
Ack.